src/pages/Send/components/PriorityField/PriorityField.stories.js

```diff
@@ -0,0 +1,98 @@
+
+/* @flow */
+
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import {
+  Form,
+  Field,
+} from 'react-final-form'
+
+import {
+  withKnobs,
+  text,
+  boolean,
+} from '@storybook/addon-knobs'
+
+import { PriorityField } from './PriorityField'
+import { InputWithUnit } from './InputWithUnit/InputWithUnit'
+
+function formStoryWrapper(
+  component,
+  extraProps = {},
+  initialValues = {},
+  validate = () => ({}),
+) {
+  return (
+    <Form
+      validate={validate}
+      initialValues={initialValues}
+      onSubmit={values => alert(JSON.stringify(values, false, 4))}
```

Comment:
Better to use [storybook action](https://www.npmjs.com/package/@storybook/addon-actions) here